index.html

```diff
@@ -21,14 +24,186 @@
+                        <li><button class="btn2">JavaScript</button></li>
+                    </ul>
+                    <div class="btn3">
+                       <h2 class="text">See this project</h2>
+                       <img class="arrow" src="img/ic_arrow_right.png">
+                    </div>
+                </div>
+            </div>
+
+            <!-- project 4 showcase -->
+            <div class="project">
+                <img class="project_img" src="img/Project4.png" alt="Project1">
+                <div class="menu">
+                    <h2 class="sub_title">Project name goes here</h2>
+                    <ul class="list">
+                        <li><button class="btn2">HTML/CSS</button></li>
+                        <li><button class="btn2">Ruby on Rails</button></li>
+                        <li><button class="btn2">JavaScript</button></li>
+                    </ul>
+                    <div class="btn3">
+                       <h2 class="text">See this project</h2>
+                       <img class="arrow" src="img/ic_arrow_right.png">
+                    </div>
+                </div>
+            </div>
+
+            <!-- project 5 showcase -->
+            <div class="project">
+                <img class="project_img" src="img/Project5.png" alt="Project1">
+                <div class="menu">
+                    <h2 class="sub_title">Project name goes here</h2>
+                    <ul class="list">
+                        <li><button class="btn2">HTML/CSS</button></li>
+                        <li><button class="btn2">Ruby on Rails</button></li>
+                        <li><button class="btn2">JavaScript</button></li>
+                    </ul>
+                    <div class="btn3">
+                       <h2 class="text">See this project</h2>
+                       <img class="arrow" src="img/ic_arrow_right.png">
+                    </div>
+                </div>
+            </div>
+
+            <!-- project 6 showcase -->
+            <div class="project">
+                <img class="project_img" src="img/Project6.png" alt="Project1">
+                <div class="menu">
+                    <h2 class="sub_title">Project name goes here</h2>
+                    <ul class="list">
+                        <li><button class="btn2">HTML/CSS</button></li>
+                        <li><button class="btn2">Ruby on Rails</button></li>
+                        <li><button class="btn2">JavaScript</button></li>
+                    </ul>
+                    <div class="btn3">
+                       <h2 class="text">See this project</h2>
+                       <img class="arrow" src="img/ic_arrow_right.png">
+                    </div>
+                </div>
+            </div>
+        </div>
+    </section>
+    <!-- work section ends -->
+
+
+    <!-- about section starts -->
+    <section class="about">
+        <div class="cv_block">
+            <h1 class="title_font">About</h1>
```

Comment:
- [ ] Kindly change this to another header tag different from `h1`.  `h1` tag is mostly use to give a clear indication of the content of the page. 😄